README.md

```diff
@@ -33,6 +33,8 @@ with:
   pr-title: "Update dependencies"
   # The update strategy, can be 'reuse', 'eager' or 'all'
   update-strategy: eager
+  # The save strategy, can 'compatible', 'wildcard', 'exact' or 'minimum'
+  save-strategy: compatible
```

Comment:
```suggestion
  save-strategy: minimum
```
Correct the default value

Reply:
Ok, will do, but how about `update-strategy` and `unconstrained`, should they also have the default values, i.e. `reuse` and `false, respectively?

Reply:
Oh sure, i didn't notice it is the example in readme